app/helpers/sessions_helper.rb

```diff
@@ -9,6 +9,10 @@ def remember user
     cookies.permanent[:remember_token] = user.remember_token
   end
 
+  def current_user? user
```

Comment:
Tìm hiểu về `SOLID principles in Ruby`,
move method này qua model `User` nhé

Reply:
trước tiên em nêu tóm tắt lại những gì đã tìm hiểu được về SOLID đã nhé :)

Reply:
![solid](https://user-images.githubusercontent.com/25588847/43059711-f8ec8f40-8e77-11e8-8000-3913ac512df3.png)

Reply:
`Chị có thể giải thích thêm giúp em tại sao nên chuyển method này sang model User không ạ?`
=> Follow nguyên tắc thiết kế số 1 (Single ...) - Mỗi class chỉ nên có một trách nhiệm duy nhất và số 4 (Interface ...) - Class không nên bị phụ thuộc vào các interface mà nó không sử dụng nhé em :D